static/js/lib/currency.js

```diff
@@ -25,6 +23,8 @@ export const excludedCurrencyCodes = [
   'XXX'
 ];
 
+const labelSort = R.sortBy(R.compose(R.toLower, R.prop('label')));
```

Comment:
Instead of this, can we make a new file (`lib/location.js`) and put both `getLocation` (from `util.js`) and `codeToCountryName` in there?